yarn.lock

```diff
@@ -12949,6 +12944,17 @@ ovh-ui-angular@^3.12.0:
     popper.js "^1.14.4"
     ui-select "^0.19.8"
 
+ovh-ui-angular@^3.14.0:
+  version "3.14.0"
```

Comment:
`develop` version is `3.15.0` (https://github.com/ovh/manager/blob/develop/yarn.lock#L12814)